Source/WebKit/UIProcess/UserMediaPermissionRequestManagerProxy.cpp

```diff
@@ -266,7 +266,7 @@ void UserMediaPermissionRequestManagerProxy::denyRequest(UserMediaPermissionRequ
     }
 
 #if ENABLE(MEDIA_STREAM)
-    page->legacyMainFrameProcess().send(Messages::WebPage::UserMediaAccessWasDenied(request.userMediaID(), toWebCore(reason), message, invalidConstraint), page->webPageIDInMainFrameProcess());
+    page->legacyMainFrameProcess().send(Messages::WebPage::UserMediaAccessWasDenied(*request.userMediaID(), toWebCore(reason), message, invalidConstraint), page->webPageIDInMainFrameProcess());
```

Comment:
Do we need a check for `request.userMediaID()` before this?

Reply:
I can add one but the previous code was already assuming this couldn't be an invalid identifier (since sending an invalid identifier over IPC would have led to crashes).